components/brave_wallet_ui/panel/container.tsx

```diff
@@ -758,6 +771,7 @@ function Container () {
               <Buy
                 onChangeBuyView={onChangeSendView}
                 selectedAsset={selectedBuyAsset}
+                onShowCurrencySelection={onShowCurrencySelection}
```

Comment:
Could do the same here?

Reply:
@Douglashdaniel The function is already using `navigateTo` function internally as show here: https://github.com/brave/brave-core/blob/c2a4f8d2b7834199a805d767fc829c80cb904aee/components/brave_wallet_ui/panel/container.tsx#L426-L428